### PR TITLE
Add EU compliant 2.4 region - limited to 10mw

### PR DESCRIPTION
**Pull request by wvarty**
unknown
unknown

### CODE REVIEW

File: src/lib/POWERMGNT/POWERMGNT.h [hunk 1]
```diff
@@ -20,6 +20,11 @@
     #define MaxPower HighPower
 #endif
 
+#if defined(Regulatory_Domain_EU_CE_2400)
+    #undef MaxPower
+    #define MaxPower PWR_10mW
+#endif
+
```
Comment (pkendall64) [inline src/lib/POWERMGNT/POWERMGNT.h:27]: I think you will need to add `MinPower` and undef `HighPower` too.
Also, problematic is that Namimno Flash and HM ES24Pro, MinPower is defined as 25mW so they have no value for 10mW

Comment (maybenikhil) [reply src/lib/POWERMGNT/POWERMGNT.h:27]: Can the Namimno Flash's amp even go down to 10mW? Or do manufacturers just have to label EU compliance (or not)

**Review by JyeSmith — APPROVED** [review]

**Review by pkendall64 — APPROVED** [review]

**Review by maybenikhil — APPROVED** [review]

File: src/python/build_flags.py [hunk 2]
```diff
@@ -151,7 +151,9 @@ def get_git_version():
 
 if "_2400" not in target_name and \
```
Comment (maybenikhil) [inline src/python/build_flags.py:152]: Could revert to old behavior:
```python
if not fnmatch.filter(build_flags, '*-DRegulatory_Domain*'):
    print_error('Please define a Regulatory_Domain in user_defines.txt')
```

**Comment by CapnBry** [comment]
The Lua item should be removed because this information will be integrated into an existing Lua item (which displays the reg domain). No need to add more code and loading time if we can condense it into another field easily.

Also, aren't the `fnmatch.filter`s backward?

**Comment by maybenikhil** [comment]
The REG domain Lua Item technically only shows the frequency but I could change it specifically for this one to show "CE" instead of the frequency

**Comment by CapnBry** [comment]
> The REG domain Lua Item technically only shows the frequency but I could change it specifically for this one to show "CE" instead of the frequency

Yeah for sure it should definitely not just say the frequency, but instead show a short reg domain. That would be the most useful and eliminate the necessary to have a second field definition.

**Review by CapnBry — APPROVED** [review]

**Comment by CapnBry** [comment]
I've updated this PR to remove the conflict and get up to date with current master, as well as clarified the build_flags regdomain tests by splitting it out into the two things it is trying to guard based on Wez's requirements and my discussion with Nikhil.

The only thing is that it isn't exactly accurate because a user can declare `Regulatory_Domain_EU_CE_2400` and expect to get 10mW but if their module only does 25mW minimum, it will still put out 25mW. Same thing with the RX, it always puts out max power, which can be up to 250mW for some RX? Therefore, this is only 10mW compliance for hardware capable of going that low, and only for RX that do not have an amp.

**Pull request merged**